### DISCUSSION TITLE
2FA

### DISCUSSION THREAD

**Discussion by nemonowa** [community/community#138864] [Other Feature Feedback, Questions, & Ideas]
リポジトリを人質に2FAを強制するのは悪質です。コード資産のセキュリティ強化の必要性は理解しますが、自分のアカウントのログアウトすら出来ないのは異常です。このような運営の強硬姿勢がエスカレートしていかないことを冀求します。

**Comment by LiteBrite82** [comment]
Hi @nemonowa , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
